src/fiatconnect/BankDetailsScreen.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { FiatAccountSchema } from '@fiatconnect/fiatconnect-types'
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import { addNewFiatAccount } from 'src/fiatconnect'
+import { Screens } from 'src/navigator/Screens'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+import BankDetailsScreen from './BankDetailsScreen'
+
+jest.mock('src/fiatconnect', () => ({
+  ...(jest.requireActual('src/fiatconnect') as any),
+  addNewFiatAccount: jest.fn(() => Promise.resolve()),
+}))
+
+const store = createMockStore({})
+const providerURL = 'https://superLegitCICOProvider.com'
```

Comment:
LOL

Reply:
```suggestion
const providerURL = 'https://superLegitCICOProvider.valoraapp.com'
```
because I'm paranoid